docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -36,11 +35,11 @@ spec:
 +
 [NOTE]
 ====
-* In a non-java application, that is a link:https://github.com/redhat-appstudio/build-definitions/blob/main/task/buildah/0.1/buildah.yaml[Buildah] task, adding the parameter mentioned above automatically isolates the build from the network, restricting it to only build from dependencies listed in your Git repository. If you need to pull in dependencies from a package manager for one of the xref:how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc#supported-languages[supported languages], see xref:how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc[Prefetching the package manager dependencies for the Hermetic build].
+* In a non-Java application that is a link:https://github.com/redhat-appstudio/build-definitions/blob/main/task/buildah/0.1/buildah.yaml[Buildah] task, setting the '**hermetic**' parameter to '**true**' automatically isolates the build from the network, which restricts it to building only from dependencies listed in your Git repository. If you need to pull in dependencies from a package manager for one of the xref:how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc#supported-languages[supported languages], see xref:how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc[Prefetching the package manager dependencies for the Hermetic build].
 
-* Do not add these parameters to the link:https://github.com/burrsutter/partner-catalog-stage/blob/e2ebb05ba8b4e842010710898d555ed3ba687329/.tekton/partner-catalog-stage-wgxd-pull-request.yaml#L87[`pipelineSpec.params`] section, as it should always display the default values for hermetic builds.
+* Do not add these parameters to the link:https://github.com/burrsutter/partner-catalog-stage/blob/e2ebb05ba8b4e842010710898d555ed3ba687329/.tekton/partner-catalog-stage-wgxd-pull-request.yaml#L87[`**pipelineSpec.params**`] section, as it should always display the default values for hermetic builds.
 ====
-. Create a pull request (PR) by committing your changes to the repository of the component.
+. Commit your changes to the component repository and create a pull request (PR).
 
 . Review and merge the PR.
```

Comment:
What do we mean by review here? Are they validating that the build triggered by the PR built (like the verification steps after this section), and then merging the PR?

Reply:
fair question, @mattreid. i assume the original writer meant something broader, like "look over", not verifying that it built. do you think users will have the same question as you? i could change it to something like, "Look over your PR and then, when you're satisfied with your changes, merge it." or "If your PR looks OK and all checks have passed, merge it." wdyt?

Reply:
When the PR is created, it should trigger a build, if that build fails, they won't want to merge the PR, since their code change breaks something in the component. If it builds successfully, and the other checks come back green, then they could merge it.

To me, it feels like these steps could directly lead into validation section, and encourage the user to merge the PR if there were no issues (or make a code change and rebuild if it failed, so they can try to get it working and then merge the PR to enable the hermetic functionality).

Reply:
@mattreid understood. let me tweak. thanks!